### ISSUE TITLE
Collapsible Listbox example: aria-labelledby on button missing from roles, states, and properties table

### ISSUE THREAD

**Issue by DavidMacDonald** [w3c/aria-practices#584]
https://www.w3.org/TR/wai-aria-practices-1.1/examples/listbox/listbox-collapsible.html

In the description of the widget there is a table with roles states and properties. It says aria-expanded is on the button but the button doesn't have that attribute. It says the aria-labelledby is on the ```<ul>``` but there is an aria-labelledby on the actual coded button that is not there in the list. It also lists activedescendant which does not appear to be on the widget.

Also the triggering button has no aria-expanded on page load, but it shows up once the button is activated. I suggest it be there as "false" on page load so the user hears that it is collapsed when first encountering it.

Also there is the sentence "or more information about this focus management technique, see Using aria-activedescendant to Manage Focus." with a link that does not go to that article.

**Comment by mcking65** [comment]
@DavidMacDonald commented:

> In the description of the widget there is a table with roles states and properties. It says aria-expanded is on the button but the button doesn't have that attribute.

It says that aria-expanded is set when the list is visible. And, it is. Activate the button and then inspect, and you will see it.

> It says the aria-labelledby is on the <ul> but there is an aria-labelledby on the actual coded button that is not there in the list.

Good catch, thank you!! I will fix this.

> It also lists activedescendant which does not appear to be on the widget.

The table states that aria-activedescendant is on the UL element. And, the first bullet of that description says, "Set by the JavaScript when it displays and sets focus on the listbox; otherwise is not present." It is definitely there; that is what is enabling screen readers to read the listbox options.

> Also the triggering button has no aria-expanded on page load, but it shows up once the button is activated. I suggest it be there as "false" on page load so the user hears that it is collapsed when first encountering it.

The rationale for not including it is similar to the rationale for not including `aria-expanded="false"` on menu buttons when the menu is not visible. See my [April 19, 2017 comment on issue 326](https://github.com/w3c/aria-practices/issues/326#issuecomment-295584897).

Net: the label, role button, and haspopup property of listbox is enough verbosity to clearly describe the button purpose, state, and interaction model in the collapsed state. Adding expanded is useful to touch users when the listbox is visible.

> Also there is the sentence "or more information about this focus management technique, see Using aria-activedescendant to Manage Focus." with a link that does not go to that article.

Yes, this is a side effect of the respec javascript that we use during publication. Respec has removed the target ID from the main document! Unfortunately, we don't have a solution to this yet, and we obviously need one!!

**Comment by mcking65** [comment]
@DavidMacDonald,

Based on my comment above, there is one problem that needed fixing. I fixed that with commit 61a19b3.
Closing as done. Please feel free to re-open if you believe the resolution warrants more discussion.
